docs/likelihood.rst

```diff
@@ -198,12 +198,14 @@ As explained in detail in :cite:`likelihood-Cranmer:1456844`, adding uncertaint
 each sample would yield a very large number of nuisance parameters with limited
 utility. Therefore a set of bin-wise scale factors :math:`\gamma_b` is
 introduced to model the overall uncertainty in the bin due to MC statistics.
-The constraint term is constructed as a set of Gaussian constraints with a
-central value equal to unity for each bin in the channel. The scales
-:math:`\sigma_b` of the constraint are computed from the individual
+The constraint term is constructed as a set of constraints with a
+central value equal to unity (e.g. `Gauss(:math:`\mu` = 1, :math:`\sigma_b`)` for each bin). The scales
```

Comment:
I will unhelpfully note before I sign off tonight that there is a Sphinx render issue here with `:math:`

![sphinx-issue](https://user-images.githubusercontent.com/5142394/166089428-635bafe9-52cf-4c7d-891e-c58a144f9f20.png)

I don't mind fixing it tomorrow though myself while reviewing. So don't take this as criticism or a demand &mdash; just a note.